changelogs/fragments/2505-proxmox_nic-set_mtu_on_interface_even_not_virtio.yml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - proxmox_nic - set mtu on interface even if it is not virtio(https://github.com/ansible-collections/community.general/pull/2505).
```

```suggestion
  - proxmox_nic - set mtu on interface even if it is not virtio (https://github.com/ansible-collections/community.general/pull/2505).
```
But since `proxmox_nic` hasn't been in a release yet, there shouldn't be a changelog fragment. So better remove this completely :)

---

I was just going to ask if this should be `minor_changes` or `bugfixes` :)
Removed in https://github.com/ansible-collections/community.general/pull/2505/commits/2f42417567d7548475c76b846147a59f5b4613e8.